lib/config/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "wasmer-config"
+version = "0.1.0"
+description = "Configuration types for Wasmer."
+edition.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+keywords = ["wasm", "wasmer", "toml"]
+categories = ["parser-implementations", "wasm"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+anyhow = "1"
+toml = "0.8"
+thiserror = "1"
+semver = { version = "1", features = ["serde"] }
+serde_json = "1"
+serde_yaml = "0.9.0"
+serde_cbor = "0.11.2"
+indexmap = { version = "2", features = ["serde"] }
```

Comment:
Can we make of this a workspace dependency?

Reply:
Done